benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1426,6 +1434,15 @@ ppx::grfx::Format GraphicsBenchmarkApp::RenderFormat()
     return renderFormat;
 }
 
+void GraphicsBenchmarkApp::CreateColorsForDrawCalls()
+{
+    // Create colors randomly.
+    mColorsForDrawCalls.resize(kMaxSphereInstanceCount);
+    for (size_t i = 0; i < kMaxSphereInstanceCount; i++) {
+        mColorsForDrawCalls[i] = float4(mRandom.Float(), mRandom.Float(), mRandom.Float(), 0.5f);
```

Comment:
set alpha to 1.0?

Reply:
0.5 is fine, the alpha will be ignored if alpha blend is disabled. it is set to 0.5 so that when alpha blending is enabled, there is no need to change the shader const